apps/web/app/(app)/gardens/[chain]/[garden]/[community]/create-pool/page.tsx

```diff
@@ -17,11 +17,10 @@ export default function CreatePool({
   params: { chain: number; garden: string; community: string };
 }) {
   const { data: result, error: error } =
-    useSubgraphQueryByChain<getPoolCreationDataQuery>(
-      chain,
-      getPoolCreationDataDocument,
-      { communityAddr: community, tokenAddr: garden },
-    );
+    useSubgraphQueryByChain<getPoolCreationDataQuery>({
```

Comment:
maybe change the name of the func to "useSubgraphQuery" since chain is not a parameter any more and it is handled inside.

Reply:
Fixed

Reply:
4acd7abbb5b217675b63f445a07a671eb9aa1607